Loop2018DataAnalysis.ipynb

```diff
@@ -498,6 +499,7 @@
     "dfs.catalog = clean_df(dfs.catalog)\n",
     "dfs.ftir = clean_df(dfs.ftir)\n",
     "dfs.reagent = clean_df(dfs.reagent)"
+    "print \"Finished cleaning 'Sample form' field at \" + str(datetime.datetime.time(datetime.datetime.now()))"    
```

Comment:
I've created a function to get the current time, and we should put brackets around print statements to make them python3 compatible. I know I've not been doing that myself, but I'll start now too :)
```suggestion
    "print (\"Finished cleaning 'Sample form' field at %s\" % now())"    
```